api/v1/controllers/collectors.js

```diff
@@ -206,6 +206,37 @@ function deregisterCollector(req, res, next) {
 } // deregisterCollector
 
 /**
+ * Reregister a collector. Access restricted to Refocus Collector only.
```

Comment:
Wondering if a user can reregister a collector.?

Reply:
What would be the use case? What are your thoughts @iamigo

Reply:
I think just a collector.